test_results/junit.xml

```diff
@@ -0,0 +1,50 @@
/><testcase classname="tests.test_prediction" name="test_default_category_pred_upload_update" time="2.475" /><testcase classname="tests.test_prediction" name="test_default_category_pred_upload_ignore" time="2.449" /><testcase classname="tests.test_prediction" name="test_mixed_pred_upload" time="4.337" /><testcase classname="tests.test_prediction" name="test_mixed_pred_upload_async" time="34.282" /><testcase classname="tests.test_prediction" name="test_mixed_pred_upload_async_with_error" time="30.156" /><testcase classname="tests.test_prediction" name="test_default_category_pred_upload_async" time="29.173" /><testcase classname="tests.test_public_datasets" name="test_get_pandaset_items" time="1.389" /><testcase classname="tests.test_scene" name="test_frame_add_item" time="0.001" /><testcase classname="tests.test_scene" name="test_scene_from_json" time="0.001" /><testcase classname="tests.test_scene" name="test_scene_property_methods" time="0.000" /><testcase classname="tests.test_scene" name="test_scene_add_item" time="0.000" /><testcase classname="tests.test_scene" name="test_scene_add_frame" time="0.000" /><testcase classname="tests.test_scene" name="test_scene_upload_sync" time="0.000"><skipped type="pytest.skip" message="Deactivated sync upload for scenes">/Users/drakewong/nucleus-python-client/tests/test_scene.py:248: Deactivated sync upload for scenes</skipped></testcase><testcase classname="tests.test_scene" name="test_scene_and_cuboid_upload_sync" time="0.000"><skipped type="pytest.skip" message="Deactivated sync upload for scenes">/Users/drakewong/nucleus-python-client/tests/test_scene.py:269: Deactivated sync upload for scenes</skipped></testcase><testcase classname="tests.test_scene" name="test_scene_upload_async" time="17.503" /><testcase classname="tests.test_slice" name="test_reprs" time="0.001" /><testcase classname="tests.test_slice" name="test_slice_create_and_delete_and_list" time="7.793" /><testcase classname="tests.test_slice" name="test_slice_create_and_export" time="2.442" /><testcase classname="tests.test_slice" name="test_slice_append" time="3.810" /><testcase classname="tests.test_slice" name="test_slice_send_to_labeling" time="0.000"><skipped type="pytest.skip" message="404 not found error">/Users/drakewong/nucleus-python-client/tests/test_slice.py:160: 404 not found error</skipped></testcase><testcase classname="tests.test_taxonomy" name="test_add_taxonomy" time="0.528" /><testcase classname="tests.test_upload_response" name="test_reprs" time="0.001" /><testcase classname="tests.test_utils" name="test_serialize" time="0.001" /><testcase classname="tests.modelci.test_eval_function" name="test_list_eval_functions" time="0.134"><failure message="nucleus.modelci.errors.EvalFunctionNotAvailableError: Eval function 'IOU' is not available to the current user. Is Model CI enabled for the user?">CLIENT = NucleusClient(api_key='test_a0453d259e5e4ef7b4b6009b441b0c24', use_notebook=False, endpoint='https://api.scale.com/v1/nucleus')
```

Comment:
doesn't seem like this file should be included in this PR

Reply:
ack, thanks! not sure how this snuck in there. maybe related to why i keep failing CI on this